testutil/integration/suites/migration.go

```diff
@@ -128,19 +127,33 @@ func (s *MigrationModuleSuite) QueryAllMorseClaimableAccounts(t *testing.T) []mi
 	return morseClaimableAcctRes.MorseClaimableAccount
 }
 
+// GetSharedParams returns the shared module params.
+func (s *MigrationModuleSuite) GetSharedParams(t *testing.T) sharedtypes.Params {
```

Comment:
I believe @red-0ne has added similar helpers recently (but potentially in a different part of the codebase).

Please just double-check that there's no overlap before merging